subsys/bluetooth/controller/hci_driver.c

```diff
@@ -684,6 +683,9 @@ static int configure_memory_usage(void)
 	}
 #endif
 
+#if defined(CONFIG_BT_PERIPHERAL) || \
+	defined(CONFIG_BT_CENTRAL) || \
+	defined(CONFIG_BT_LL_SOFTDEVICE_MULTIROLE)
```

Comment:
Do we need the `defined(CONFIG_BT_LL_SOFTDEVICE_MULTIROLE)`?

Reply:
No, I dont think so. Removed it now.